sdcflows/data/sd_syn_sloppy.json

```diff
@@ -1,25 +1,25 @@
 {
     "collapse_output_transforms": true,
-    "convergence_threshold": [ 1e-06, 1e-06, 1e-08 ],
-    "convergence_window_size": [ 5, 3, 2 ],
+    "convergence_threshold": [ 1e-06, 1e-08 ],
+    "convergence_window_size": [ 5, 2 ],
     "dimension": 3,
     "interpolation": "Linear",
-    "metric": [ "Mattes", "CC", "CC" ],
-    "metric_weight": [ 1, 1, 1 ],
-    "number_of_iterations": [ [ 10, 0 ], [ 10, 5, 0 ], [ 2 ] ],
+    "metric": [ ["Mattes", "Mattes"], ["Mattes", "Mattes"] ],
+    "metric_weight": [ [0.5, 0.5], [0.5, 0.5] ],
+    "number_of_iterations": [ [ 20, 10 ], [ 2 ] ],
```

Comment:
Same as `sd_syn`, but with factor of 0.1 and 0.2 applied, matching the previous factors 0.1, 0.5, 0.2.